packages/application-components/src/components/dialogs/info-dialog/README.md

```diff
@@ -0,0 +1,41 @@
+# InfoDialog
+
+## Usage
+
+```js
+import { InfoDialog } from '@commercetools-frontend/application-components';
+```
+
+#### Description
+
+Info dialogs are controlled components used to show more information about a particular feature, UI element, etc. They are mainly composed of text paragraphs.
+
+#### Usage
+
+```js
+<InfoDialog
+  title={<FormattedMessage {...messages.infoTitle} />}
+  isOpen={isOpen}
+  onClose={handleClose}
+>
+  <Spacings.Stack scale="m">
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+  </Spacings.Stack>
+</InfoDialog>
+```
+
+#### Properties
+
+| Props                  | Type       | Required | Values   | Default | Description                                                                                                                 |
+| ---------------------- | ---------- | :------: | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `title`                | `node`     |    ✅    | -        | -       | The title of the Info Dialog                                                                                                |
+| `isOpen`               | `boolean`  |    ✅    | -        | -       | Indicates whether the dialog should open or not                                                                             |
+| `onClose`              | `function` |    ✅    | -        | -       | Called when the dialog closes (click on overlay, click on close button, press ESC)                                          |
+| `closeTimeoutMS`       | `number`   |    -     | -        | `150`   | The timeout in ms for the dialog to close                                                                                   |
```

Comment:
can we rename this to `closeAfter`?

Reply:
Sure (the name is the same as the prop in `react-modal` btw)

Reply:
Yeah I saw that. I figured that we aren't married to the idea of keeping our component in sync with `react-modal`. Maybe in the future we migrate aware from `react-modal`. It makes more sense to keep our props consistent